server/src/main/java/org/elasticsearch/action/admin/indices/get/GetIndexRequest.java

```diff
@@ -99,6 +98,8 @@ public static Feature fromId(byte id) {
     private static final Feature[] DEFAULT_FEATURES = new Feature[] { Feature.ALIASES, Feature.MAPPINGS, Feature.SETTINGS };
     private Feature[] features = DEFAULT_FEATURES;
     private boolean humanReadable = false;
+    private boolean flatSettings = false;
+    private boolean includeDefaults = false;
```

Comment:
the point that you make on these not being serializable is a good one. I think that is a good choice, if we want to make it more explicit we could mark these `transient`